src/hypofuzz/dashboard.py

```diff
@@ -189,6 +210,7 @@ async def api_pycrunch_file(request: Request) -> Response:
     Route("/api/tests/", api_tests),
     Route("/api/tests/{node_id:path}", api_test),
     Route("/api/patches/", api_patches),
+    Route("/api/collected_tests/", api_collected_tests),
```

Comment:
(for later) this makes me think we should have some API documentation, or maybe just point users at a [schema](https://www.starlette.io/schemas/)?